### ISSUE TITLE
Character sheet: Conditions

### ISSUE THREAD

**Issue by Hiplobbe** [YanKlInnomme/FoundryVTT-k4lt#29]
One thing missing from the character sheets are the conditions sections. (image attached)

The hard part about this is that the negative numbers are not always applicable to the roll.  I would suggest that there is a checkbox that the player can check if it should apply to the roll. 

![{2FEDCD47-9E0F-45DB-8340-EA43AB25A2CA}](https://github.com/user-attachments/assets/8453c660-3d28-4997-8c44-90fe25f8bb74)

**Comment by YanKlInnomme** [comment]
Except this is wrong, here is the official content of the book:

![image](https://github.com/user-attachments/assets/6900d9a2-cfba-48dd-9d34-1d18debb5089)

**Comment by Hiplobbe** [comment]
You might've missunderstood me, I meant that having a section where you keep track of the conditions that a character have accumulated. And being able to apply them to roll, where applicable, currently I do not see anyway to do that with the correct character sheet.

**Comment by YanKlInnomme** [comment]
In the official rules of Kult: Divinity Lost, the "conditions" mentioned in your example do not appear anywhere. They may come from a homebrew variant, a personal interpretation, or a third-party source, but they are not part of the official mechanics.

To clarify, here’s what the official rules state:
When a character attempts to Keep it Together, the result of the roll may lead to specific narrative or mechanical effects. Depending on the roll outcome:

On a partial success (10–14), the player chooses a reaction from several possible consequences (e.g., becoming angry, scared, or obsessed). These choices are narrative and may sometimes include temporary modifiers, such as a penalty to Stability or a change in relationships.

On a failure (9 or less), the consequences are more severe, often involving larger penalties or even lasting trauma.

However, the game does not include a dedicated section on the character sheet to track "accumulated conditions" as a persistent mechanic. These effects are temporary or handled narratively. Adding such a feature to the character sheet would fall under a homebrew adaptation rather than official rules.

**Comment by Hiplobbe** [comment]
Oh okey, I understand your issue. However I feel that you should not put any weight on what conditions are on the image provided and more focus on the feature itself (being able to keep track of the conditions).

"These effects are temporary or handled narratively. Adding such a feature to the character sheet would fall under a homebrew adaptation rather than official rules.", well this is only because the nature of pen and paper, and constantly adding and removing would destroy the paper. Working with a digital media removes this issue. 

Players and GM do have a need for a way to keep track of the conditions applied to a character, if not on the character sheet it will be put somewhere else. No one wants to keep track of this in their head.

**Comment by YanKlInnomme** [comment]
I understand your interest in tracking conditions more systematically, but I’m struggling to see the need for accumulating penalties in this way.

In Kult: Divinity Lost, taking a condition during a Keep it Together roll already results in a Stability loss, which directly translates into penalties (or multiple penalties, depending on the Stability level) for future rolls. Adding another layer of penalties on top of this would make the game even more punitive—and Kult is already harsh enough by design.

The system is designed to reflect the immediate and evolving psychological impact of events, not to burden characters with accumulating maluses. In this sense, adding a feature to track these "conditions" would still represent a departure from the original intent of the rules.

**Comment by Hiplobbe** [comment]
Looking at the rules it is unclear if you get a -1 on rolls or just -1 stability, or BOTH. The creator of the roll20 sheet that i referenced seem to assume that it is both, since "distracted" gives a -2 but not stability loss.

If we assume that you get both (which it might well be because being broken is a big part of the system), some way of keeping track of conditions would be beneficial.

With "You get −1 in situations where this condition would be a hindrance to you.", we can assume that the conditions do in fact also give minuses to rolls, as well as giving minus in stability (if such a condition is chosen).

**Comment by YanKlInnomme** [comment]
Thank you for your response. I understand where the confusion might come from, so let me clarify based on the rules. I am referencing the French translation, which, in my opinion, provides a clearer explanation on this point:

Translated, this means:
"In all situations where the character’s state of mind (condition) hinders them, they suffer a -1 penalty on all relevant rolls, except for Confusion, which imposes a -2. The state of mind can be resolved once the character has had the opportunity to process the trauma, typically after a period without stress or through therapy."

This indicates that conditions indeed impose penalties on relevant rolls in addition to Stability loss. However, these penalties are situational and narrative, meaning they don’t universally apply to all rolls. Instead, they are tied to specific contexts where the condition is narratively relevant. This ensures the game remains focused on psychological tension without becoming overly punitive or bogged down by accumulating maluses.

In this sense, the penalty is not ongoing but rather forward, applying only when relevant to the situation. With this in mind, I doubt the usefulness of having a dedicated checkbox section for conditions on the character sheet (to check a box that will simply be unchecked after the next roll). Personally, I just use the Forward input at the top of the character sheet to apply penalties when the situation calls for it.

**Comment by Hiplobbe** [comment]
Yes, my point being that it has to be written down somewhere, since it is a "debuff" that is set on the character. To automatically or semi-automatically put this on the rolls is actually secondary to the feature itself.  

Just like my issue (https://github.com/YanKlInnomme/FoundryVTT-k4lt/issues/28) bare minimum would be to have a free text section where you can write this down. But preferably have some sort of checkboxes and being able to apply them to rolls would be neat but not "necessary" for the players to keep track of the conditions they've accumulated.

**Comment by YanKlInnomme** [comment]
From my perspective, since conditions in Kult: Divinity Lost are temporary and situational, they can already be noted in the existing free-text section or using the Forward input for specific penalties. However, I see the value of a dedicated space if it allows for adding contextual details.

For example, for Angry, a field could be added to specify who or what the character is angry at. For Scared, a space to indicate the source of the fear. This would provide players with better visibility and enhance immersion by contextualizing these states of mind.

I’m considering exploring this idea further to see how it could be implemented while respecting the game’s narrative intent. What do you think?

**Comment by Hiplobbe** [comment]
Absolutely, thank you for being understanding. :)

**Comment by YanKlInnomme** [comment]
![Capture d'écran 2025-01-16 092327](https://github.com/user-attachments/assets/1ff4e22c-df6b-4f6a-93bf-2ab53e1668a0)
